flink-table-store-connector/src/test/java/org/apache/flink/table/store/connector/sink/global/GlobalCommitterOperatorTest.java

```diff
@@ -166,16 +166,33 @@ public void endOfInput() throws Exception {
         assertThat(globalCommitter.getCommittedData()).contains("elder+patience+silent");
     }
 
+    @Test
+    public void streamingNoEndOfInput() throws Exception {
+        final DefaultGlobalCommitter globalCommitter = new DefaultGlobalCommitter();
+
+        final OneInputStreamOperatorTestHarness<CommittableMessage<String>, Void> testHarness =
+                createTestHarness(globalCommitter, false);
+        testHarness.initializeEmptyState();
+        testHarness.open();
+        List<String> input = Arrays.asList("silent", "elder", "patience");
+        testHarness.processElements(committableRecords(input));
+        testHarness.endInput();
+        testHarness.close();
+        assertThat(globalCommitter.getCommittedData()).isEmpty();
```

Comment:
Nit: line#150 `assertThat(globalCommitter.getCommittedData().isEmpty()).isTrue();` can be shorten as `assertThat(globalCommitter.getCommittedData().isEmpty())`. Since GitHub does not allow comments on unchanged lines, I post here.

Reply:
What do you mean? `assertThat` always needs be invoked return class.

Reply:
Sorry, I'm typo, I mean `assertThat(globalCommitter.getCommittedData()).isEmpty();`